src/components/image-block/react/ImageBlock.tsx

```diff
@@ -42,8 +42,8 @@ interface IImageBlockProps extends IGenericProps {
     fillHeight?: boolean;
     /* The url of the image we want to display in the image-block. */
     image: string;
-    /** A list of tags, those tags will be displayed in a chip component. */
-    tags?: string[];
+    /* Tags elements to be transcluded into the component */
```

Comment:
Could you replace the `/*` comments with `/**` jsdoc comments in this interface and the avatar props interface?
Normal comments are not detected by the program generating the properties table on the demo site.
You can make sur the property tables appear by running with `yarn start:react` and naviguating to the avatar and imageblock demo page (in product > components)